.cirrus.yml

```diff
@@ -448,6 +446,33 @@ osx_alt_build_task:
     always: *runner_stats
 
 
+# Build freebsd release natively on a FreeBSD VM.
+freebsd_alt_build_task:
+    name: "FreeBSD Cross"
+    alias: freebsd_alt_build
+    # Docs: ./contrib/cirrus/CIModes.md
+    only_if: *no_multiarch_rhel
+    depends_on:
+        - build
+    env:
+        <<: *stdenvars
```

Comment:
Including these could cause some big confusion if a maintainer is glancing at the logs.  It may lead them (or CI scripts) to mistakenly treat this context as being Fedora.  Consider either not referencing this alias, or overwriting at least `$DISTRO_NV`, `$VM_IMAGE_NAME`, and maybe blank out`$CTRFQIN`.

These are all mainly used in `setup_environment.sh` (which isn't in use for this task), and some parts of `lib.sh` that probably aren't called (I didn't check).  However, they will still show up if somebody clicks 're-run in terminal' on a failed task, or expands the list in the Cirrus WebUI.

Reply:
I overwrite DISTRO_NV, VM_IMAGE_NAME, CTR_FQIN with placeholder values. Note: the osx_alt_build task also has this problem.

Reply:
> Note: the osx_alt_build task also has this problem.

Ohh good catch, you're right.  I'll fix that.

Reply:
Opened https://github.com/containers/podman/pull/19099